GCS/probe0/probe0.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
```

Comment:
The other protos use proto3 syntax. Why is this using proto2?

Reply:
Proto3 is almost fully backwards compatible with proto2, but it doesn’t allow for the proto2 Extend which we makes use of here in line 17 - extend cloudprober.probes.ProbeDef. Do you know of any way to make extend work in proto3?
reference: https://developers.google.com/protocol-buffers/docs/proto3#customoptions (proto2)
https://developers.google.com/protocol-buffers/docs/proto3#customoptions (proto3)

Reply:
The explanation makes sense. I would add it as a comment for future readers.

Reply:
will do thank you!